Riot/Modules/Room/ParticipantsInviteModal/ContactsPicker/ContactsPickerViewModel.swift

```diff
@@ -125,7 +125,11 @@ class ContactsPickerViewModel: NSObject, ContactsPickerViewModelProtocol {
         }
         
         contactsViewController.showSearch(true)
+#if HIDE_MATRIX_ID
```

Comment:
I don't want to add a new macro here, I would prefer to link the placeholder with the property `forceMatrixIdInDisplayName` defined just above
(let see our discussion to link this property with a new BuildSettings in Element)